main.go

```diff
@@ -37,22 +37,29 @@ func main() {
 
 	var tags []string
 
-	switch repo {
-	case "hub.docker.com":
+	switch {
+	case strings.Contains(repo, "hub.docker.com"):
 		t, err := retrieveFromDockerHub(image)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			os.Exit(1)
 		}
 
 		tags = t
-	case "quay.io":
+	case strings.Contains(repo, "quay.io"):
 		t, err := retriveFromQuay(image)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			os.Exit(1)
 		}
 
+		tags = t
+	case strings.Contains(repo, "amazonaws.com"):
+		t, err := retrieveFromECR(image)
+		if err != nil {
+			fmt.Println(err)
```

Comment:
You should exit with error.

```suggestion
			fmt.Fprintln(os.Stderr, err)
			os.Exit(1)
```

Reply:
This is a remaining temporary code, sorry. I fixed.